product-service/handlers/setProduct.ts

```diff
@@ -0,0 +1,55 @@
+// import { APIGatewayProxyHandler } from 'aws-lambda';
+import { uuid } from 'uuidv4';
+import 'source-map-support/register';
+import { Client } from 'pg';
+import { dbOptions } from '../utils/dbOptions';
+import { corsHeaders } from '../utils/corsHeaders';
+
+export const setProduct = async event => {
+  console.log('Add new product: ', event.body);
+
+  const client = new Client(dbOptions);
+  await client.connect();
+  const { description, price, title, image, count } = JSON.parse(event.body.replace(/'/g, "''"));
+
+  if (!description || !price || !title || !count) {
```

Comment:
this kind of manual validation is ok, but it would be better to use some advanced tools, like https://joi.dev/api/
Because there might be some problems: for example, the user can send string values for `price` and `count` and there will be unhandled errors